sw/otbn/code-snippets/rsa_verify.s

```diff
@@ -269,6 +279,8 @@ compute_rr:
     bn.sid    x8, 0(x18++)
     addi      x8, x8, 1
 
+  nop
```

Comment:
Just for my understanding: Why do we need a `nop` here after this change?

Reply:
That is just a quirk of OTBN, it can't (for the sake of reduced HW complexity) return from a subroutine just after the last statement within a loop body